app/javascript/dashboard/i18n/locale/en/teamsSettings.json

```diff
@@ -0,0 +1,123 @@
+{
+  "TEAMS_SETTINGS": {
+    "NEW_TEAM": "Create new team",
+    "HEADER": "Teams",
+    "SIDEBAR_TXT": "<p><b>Teams</b></p> <p>Organizing agents into teams let you use chatwoot the way your company is structured or to  segment the agents by their responsibilities. <br /> With Chatwoot you can create multiple teams and can have any number of agents, also a user can be part of multiple teams. <br/> You can assign conversations in chatwoot to a team so that it becomes a shared responsibility. </p>",
```

Comment:
Note: This needs to be changed, I will provide the updated text to be used here.